tests/drivers/fprotect/negative/dummy_huk.hex

```diff
@@ -0,0 +1,3 @@
+:02000004000FEB
```

Comment:
I see this file used in the associated `CMakeLists.txt`, but do we normally commit hex files for tests?

Reply:
This commit is removed since it requires another PR. The reason for commiting the hex file is for convenience. It's the easiest way to place the data at that location in flash. I think this is unproblematic since it builds out of the box, and it's only a test.